platform/include/logging_stack.h

```diff
@@ -0,0 +1,93 @@
+/*
```

Comment:
This is a example/demo logging implementation - think about if it is the right location for this file.

Reply:
Good point. How about `demos/logging-stack`?

Reply:
Moved all logging related files to `demos/logging-stack`. Still have some implementation files in `platform` which I will move to `demos/logging-stack` in a follow-up PR.

Reply:
Do you intend to keep this file or delete it?

Reply:
Deleted